osmnx/utils.py

```diff
@@ -198,6 +203,92 @@ def euclidean_dist_vec(y1, x1, y2, x2):
     return distance
 
 
+def get_unique_nodes_ordered_from_way(way_edges_df):
```

Comment:
Can you move this new function to geo_utils module? Since the refactor, that module is now holding all the network/spatial utility functions.

Reply:
done.